quakemigrate/signal/local_mag/amplitude.py

```diff
@@ -325,11 +325,14 @@ def get_amplitudes(self, event, lut):
                 else:
                     filter_sos = None
 
-                windows, picked = self._get_amplitude_windows(station, i,
-                                                              event, p_ttimes,
-                                                              s_ttimes,
-                                                              lut.fraction_tt)
-                amps[14] = picked
+                try:
+                    windows, picked = self._get_amplitude_windows(station, i,
+                        event, p_ttimes, s_ttimes, lut.fraction_tt)
```

Comment:
```suggestion
                    windows, picked = self._get_amplitude_windows(
                        station, i,
                        event, p_ttimes, s_ttimes, lut.fraction_tt)
```

Reply:
Not this?
```suggestion
                    windows, picked = self._get_amplitude_windows(
                        station, i, event, p_ttimes, s_ttimes, lut.fraction_tt)
```

Reply:
Checked on an online tool and this passes the checks (need to install flake8 locally..)

Reply:
Yep, that's also fine! My b.